source/reference/json/cloud-backup-schedule-config-file.txt

```diff
@@ -0,0 +1,241 @@
+.. _atlas-cli-cloud-backup-schedule-config-file:
+
+========================================
+Cloud Backup Schedule Configuration File
+========================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can use a cloud backup schedule configuration file to specify the
+settings required when you update a cloud backup schedule using the
+{+atlas-cli+}. The {+atlas-cli+} accepts ``.json`` cloud backup schedule
+configuration files.
+
+Use the following resources to:
+
+- Learn the :ref:`settings <cloud-backup-schedule-settings>`
+  you can specify in the cloud backup schedule configuration file.
+- View and copy :ref:`sample configuration files
+  <example-cloud-backup-schedule-config-file>`.
+
+.. _cloud-backup-schedule-settings:
+
+Cloud Backup Schedule Settings
+------------------------------
+
+|service| allows you to specify the following settings to update a cloud
+backup schedule with the {+atlas-cli+}. You can specify these cloud backup
+schedule settings either in the configuration file or as flags in the
+command:
+
+.. list-table:: 
+   :header-rows: 1 
+   :widths: 20 10 70 
+
+   * - Field 
+     - Type 
+     - Description 
+
+   * - ``autoExportEnabled``
+     - boolean
+     - Flag that indicates whether MongoDB Cloud automatically exports
+       cloud backup snapshots to the |aws| bucket.
+
+   * - ``copySettings``
+     - array
+     - List that contains a document for each copy setting item in the
+       desired backup policy.
+
+   * - ``copySettings.``
+       ``cloudProvider``
+     - string
+     - Human-readable label that identifies the cloud provider that
+       stores the snapshot copy.
+
+   * - ``copySettings.``
+       ``frequencies``
+     - array of strings
+     - List that describes which types of snapshots to copy.
+
+   * - ``copySettings.``
+       ``regionName``
+     - string
+     - Target region to copy snapshots belonging to ``replicationSpecId``
+       to. Please supply the 'Atlas Region' which can be found under
+       `Cloud Providers <https://www.mongodb.com/docs/atlas/reference/cloud-providers/>`__
+       'regions' link. 
+
+   * - ``copySettings.``
+       ``replicationSpecId``
+     - string
+     - Unique 24-hexadecimal digit string that identifies the
+       replication object for a zone in a cluster. For global clusters,
+       there can be multiple zones to choose from. For sharded clusters
+       and replica set clusters, there is only one zone in the cluster.
+       To find the Replication Spec Id, do a GET request to Return One
+       Cluster in One Project and consult the replicationSpecs array
+       :oas-atlas-op:`Return One Cluster in One Project
```

Comment:
Question: This seems odd chillen at the end here. Can we make this an inline link on the same text above?